coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -241,7 +241,7 @@ void setup_nrnthreads_on_device(NrnThread* threads, int nthreads) {
                 d_displ = (int*) acc_copyin(nrb->_displ, sizeof(int) * (nrb->_size + 1));
                 acc_memcpy_to_device(&(d_nrb->_displ), &d_displ, sizeof(int*));
 
-                d_nrb_index = (int*) acc_copyin(nrb->_nrb_index, sizeof(int) * (nrb->_size + 1));
+                d_nrb_index = (int*) acc_copyin(nrb->_nrb_index, sizeof(int) * nrb->_size);
```

Comment:
Can you briefly explain what is behind this change?

Reply:
The change is to make it match https://github.com/BlueBrain/CoreNeuron/blob/2c51992f4548210180909dff30e3f8b9a292b289/coreneuron/io/phase2.cpp#L461
without the change then `acc_copyin` reads 4 undefined bytes at the end (Valgrind complained).